internal/node/client_pool.go

```diff
@@ -6,32 +6,37 @@ import (
 	"github.com/threefoldtech/zos/pkg/rmb"
 )
 
-type NodeClientCollection interface {
+// NodeClientGetter is an interface for node client
+type NodeClientGetter interface {
 	GetNodeClient(sub subi.SubstrateExt, nodeID uint32) (*NodeClient, error)
 }
+
+// NodeClientPool is a pool for node clients and rmb
 type NodeClientPool struct {
-	nodeClients map[uint32]*NodeClient
-	rmb         rmb.Client
+	clients map[uint32]*NodeClient
+	rmb     rmb.Client
 }
 
+// NewNodeClientPool generates a new client pool
 func NewNodeClientPool(rmb rmb.Client) *NodeClientPool {
 	return &NodeClientPool{
-		nodeClients: make(map[uint32]*NodeClient),
-		rmb:         rmb,
+		clients: make(map[uint32]*NodeClient),
+		rmb:     rmb,
 	}
 }
 
-func (k *NodeClientPool) GetNodeClient(sub subi.SubstrateExt, nodeID uint32) (*NodeClient, error) {
-	cl, ok := k.nodeClients[nodeID]
+// GetNodeClient gets the node client according to node ID
+func (p *NodeClientPool) GetNodeClient(sub subi.SubstrateExt, nodeID uint32) (*NodeClient, error) {
+	cl, ok := p.clients[nodeID]
```

Comment:
lines can be merged together if cl, ok:= ; ok { return cl, nil}

Reply:
cl is used below